docs/src/main/paradox/scala/http/routing-dsl/rejections.md

```diff
@@ -1,7 +1,7 @@
 # Rejections
 
-In the chapter about constructing @ref[Routes](routes.md) the `~` operator was introduced, which connects two routes in a way
-that allows a second route to get a go at a request if the first route "rejected" it. The concept of "rejections" is
+In the chapter about constructing @ref[Routes](routes.md) the @scala[`~` operator]@java[`RouteDirectives.route()` method] was introduced, which connects two or more routes in a way
```

Comment:
The `routing-dsl/routes.md` page doesn't mention `RouteDirectives.concat`. It is mentioned in docs/src/main/paradox/scala/http/routing-dsl/directives/index.md (#1081).